draft-selander-ace-ake-authz.md

```diff
@@ -476,142 +478,37 @@ V receives the voucher response from W over the secure connection. If the receiv
 
 V verifies CRED_I and that U is an admissible device and then continues the EDHOC processing, or else discontinues the protocol.
 
-# ACE Profile
+# REST Interface at W
 
-The messages specified in this document may be carried between the endpoints in various protocols. This section defines an embedding as a profile of the ACE framework (see Appendix C of {{I-D.ietf-ace-oauth-authz}}).
+The interaction between V and W is enabled through a RESTful HTTPS interface exposed by W.
+V MUST perform a TLS handshake with W, reachable at the URI specified in LOC_W.
+V MUST use the TLS client authentication to authenticate to W, using the certificate containing the PK_V public key.
 
-* U plays the role of the ACE Resource Server (RS).
+## HTTP URIs
 
-* V plays the role of the ACE Client (C).
+W MUST support the use of the path-prefix "/.well-known/", as defined in {{RFC5785}}, and the registered name "ake-authz".
+A valid URI thus begins with "https://www.example.com/.well-known/ake-authz".
```

Comment:
Could go under /.well-known/brski

Reply:
... or /.well-known/edhoc

I think we can keep ake-authz for the time being.